pymc_marketing/clv/models/basic.py

```diff
@@ -57,7 +57,8 @@ def _add_fit_data_group(self, data: pd.DataFrame) -> None:
                 category=UserWarning,
                 message="The group fit_data is not defined in the InferenceData scheme",
             )
-            assert self.idata is not None
+            if self.idata is None:
+                raise ValueError("idata is not defined")
```

Comment:
I don't like these, because they are just a work-around stupid mypy. We know that this method can only be called after we already checked idata exists

Reply:
Reverted  https://github.com/pymc-labs/pymc-marketing/commit/f9dc0dc2371a3375ddc094d36d26ede25a665cf0 :)